doc/Comments.xml

```diff
@@ -327,10 +327,10 @@ Resets the current level to 0.
 <Index Key="@EndExample"><C>@EndExample</C></Index>
 <Heading><C>@BeginExample</C> and <C>@EndExample</C></Heading>
 <C>@BeginExample</C> inserts an example into the manual. The syntax for examples is different from
-GAPDocs example syntax in order to have a
-file that contains the example and is GAP readable. To achieve this, GAP commands are not preceded by a comment,
+&GAPDoc;'s example syntax in order to have a
+file that contains the example and is &GAP; readable. To achieve this, &GAP; commands are not preceded by a comment,
```

Comment:
Of course that text still isn't really great... A casual reader won't know what "GAPDocs example syntax" even refers to. A precise reference to what is meant (presumably the `<Example>` element?), in the form of a `<Ref ... BookName="gapdoc"...` would be much better.